vulnerabilities/static/css/font-awesome.css.ABOUT

```diff
@@ -0,0 +1,12 @@
+about_resource: font-awesome.css
+version: 4.7.0
+download_url: https://raw.githubusercontent.com/FortAwesome/Font-Awesome/v4.7.0/css/font-awesome.css
+
+name: font-awesome.css 
+homepage_url: https://github.com/FortAwesome/Font-Awesome
+owner: FortAwesome
+author: Dave Gandy
+notes: This is the minified version of Font Awesome.
+
+license: MIT
```

Comment:
All our licenses are ScanCode keys and are lowercase (e.g. case does not matter and is always normalized to lower).
Also based on the .LICENSE below it looks like the license may be rather: `mit  AND ofl-1.1 AND cc-by-4.0` and we would also need the text of each of these too.

Reply:
According to https://github.com/FortAwesome/Font-Awesome/tree/v4.7.0#license I'm pretty sure we won't need `AND` . 

Using scancode keys now  .